### PR TITLE
Feat: Added Images

### PR DESCRIPTION
**Pull request by sadathshariff**
Added
1. Round Image
2. Square Image
3.Responsive Image

### CODE REVIEW

File: docs/images/image.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+              <li><a href="../images/image.html">Images</a></li>
+            </ul>
+          </div>
+          <!-- Main content -->
+          <div class="component-container">
+            <h1 class="component-title">Images</h1>
+            <div class="component-desc">
+                
+            </div>
+
```
Comment (nishasen) [inline docs/images/image.html]: Remove the extra space here

File: docs/images/image.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                  </code>
+                 </pre>
+              </div> 
+              
+             
+            <!-- Responsive Image -->
+              <div class="component-heading">Responsive Image</div>
+              <div class="component-subHeading">
+                
+              </div>
```
Comment (nishasen) [inline docs/images/image.html]: No content has been put in this div

File: docs/images/image.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                <div class="img-wrapper">
+                    <img class="resp-img" src="https://images.unsplash.com/photo-1642186132417-675a005d259e?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwxfDB8MXxyYW5kb218MHx8fHx8fHx8MTY0MzgwMzYxMw&ixlib=rb-1.2.1&q=80&utm_campaign=api-credit&utm_medium=referral&utm_source=unsplash_source&w=1080" alt="Burj-khalifa 3D-image">
+                </div> 
+              </div>
+                <div class="example-code">
+                <pre>
+                  <code class="html hljs xml">
+                   <img class="resp-img" src="Paste image url" alt="">
+                  </code>
+                </pre>
```
Comment (nishasen) [inline docs/images/image.html:83]: Try using prettier for indentation

**Review by nishasen — CHANGES_REQUESTED** [review]
Please look into the comments

File: docs/images/image.html (hunk 2 above)
Comment (sadathshariff) [reply docs/images/image.html]: Thanks Nisha, Will look into that

**Review by nishasen — APPROVED** [review]

File: CSS/Components/images.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  height: 200px;
+  object-fit: cover;
+}
+
+.square-img {
+  border-radius: 5px;
+  width: 200px;
+  height: 200px;
+  object-fit: cover;
+}
```
Comment (Sameony) [inline CSS/Components/images.css:11]: you can create a separate img class for defining repeating properties
such as:
`.images{
  width: 200px;
  height: 200px;
  object-fit: cover;
}`

**Review by Sameony — COMMENTED** [review]
Suggesting minor adjustments

File: CSS/Components/images.css (hunk 4 above)
Comment (sadathshariff) [reply CSS/Components/images.css:11]: Thanks, will look into that

**Comment by sadathshariff** [comment]
> Suggesting minor adjustments

made the changes can you look into that

**Review by Sameony — APPROVED** [review]

**Pull request merged**